theme/style.less

```diff
@@ -211,15 +251,16 @@ header {
   }
   #volume {
     display: inline-block;
-    font-size: 0; 
+    font-size: 0;
   }
 
   #progress {
     display: flex;
     width: 100%;
   }
-  
- #progress-bar {
+
+  #progress-bar {
+    width: 100%;
```

Comment:
Only change to progress-bar, which seems to be broken. Needs testing before merging PR.